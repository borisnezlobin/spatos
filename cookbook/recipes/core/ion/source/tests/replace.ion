echo $replace("onetwo one three" "one" "1")
echo $replace("abc" "a" "")
