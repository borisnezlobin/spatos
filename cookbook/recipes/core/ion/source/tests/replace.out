1two 1 three
bc
